snippets/react-snippets.code-snippets

```diff
@@ -64,6 +64,16 @@
         ],
         "description": "Add `role='article'` to a node that can easily stand on its own separate from the main content of the page. An example is a newspaper article or a forum post. Use `aria-posinset` to indicate of the article within the feed. Use `aria-setsize` to indicate how many articles are in the feed."
     },
+    "BannerAria":{
+        "prefix": [
+            "BannerAria",
+        ],
+        "body":[
+            "role='banner'",
+            "tabIndex={0}"
```

Comment:
no need to add a tabIndex on this aria role

`tabIndex='0'` makes components focusable typically we add tabIndex to form elements like select, inputs to allow users who navigate using the keyboard to be able to interact with the element.
Since banners are typically containers for other elements we do not need to add tabIndex.

Reply:
OK Sure! Then I am removing tabIndex=0 and adding banner aria to readme.